### DISCUSSION TITLE
Aesthetic problem with backgroundless logos on GitHub organization profile pages

### DISCUSSION THREAD

**Discussion by Steellgold** [community/community#59431] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/51505384/16c44f01-7048-401e-bb9b-f7a65aa8baa8)
![image](https://github.com/community/community/assets/51505384/d9cca0b6-f858-427d-9ccc-720fd6b078f1)

Hello, I would like to report an accessibility issue on GitHub organization profile pages. When you put a logo without a background, GitHub automatically adds a background to it and this makes the logo less visible. I think you should either remove this background or add an option to choose whether you want a default background or not. Thank you for your attention.

**Comment by moonfury-ops** [comment]
Thank you

**Comment by cehfisher** [comment]
Hello @Steellgold - thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Hello @Steellgold - I wanted to follow-up on this post and point you to this resource to [specify theme context for images in Markdown](https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown). I hope this helps resolve your issue!
